### PR TITLE
add pg sqlInjection sink

### PR DESCRIPTION
**Pull request by pmcclory-contrast**

### CODE REVIEW

File: lib/sinks/sqlInjection/index.js [hunk 1]
```diff
@@ -3,3 +3,9 @@
 module.exports = {};
 module.exports['sequelize.prototype.query'] = require('./sequelize');
 module.exports['mysql/lib/Connection.query'] = require('./mysql');
+module.exports[
+  'pg.Connection.prototype.query (String)'
+] = require('./pg_string');
```
Comment (tough-griff) [inline lib/sinks/sqlInjection/index.js]: Can we nest `pg`-related in a `pg` folder?

i.e.
```suggestion
] = require('./pg/string');
```

File: lib/sinks/sqlInjection/index.js [hunk 2]
```diff
@@ -3,3 +3,9 @@
...
+module.exports[
+  'pg.Connection.prototype.query (Object)'
+] = require('./pg_object');
```
Comment (tough-griff) [inline lib/sinks/sqlInjection/index.js]: see above,

```suggestion
] = require('./pg/object');
```

File: lib/sinks/sqlInjection/pg_client.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const pg = require('pg');
+const hooker = require('hooker');
+const escape = require('escape-html');
+
+// Prevent the query method from requiring a real database connection.
+hooker.hook(require('pg/lib/client').prototype, 'query', {
```
Comment (tough-griff) [inline lib/sinks/sqlInjection/pg_client.js]: Can we access `pg/lib/client` from the `pg` object required above? is it different from `pg.Client.prototype`?

File: lib/sinks/sqlInjection/index.js (hunk 1 above)
Comment (pmcclory-contrast) [reply lib/sinks/sqlInjection/index.js]: 👍

**Review by tough-griff — CHANGES_REQUESTED** [review]
minor organizational nitpicks

**Review by tough-griff — APPROVED** [review]
Changes look good!

**Pull request merged**